knowage-vue/src/modules/documentExecution/dossier/Dossier.routes.js

```diff
@@ -1,6 +1,6 @@
 let routes = [
     {
-        path: '/dossier/:id',
+        path: '/dossier-temp/:id',
```

Comment:
Is this route needed or just a test typo?

Reply:
It was conflicting with new routes, i guess it is not needed anymore, should I remove this file completely?

Reply:
Execution and parameter sidenav have a lot of logic/formatting inside them, should I put something in separate files? @Redjaw

Reply:
> It was conflicting with new routes, i guess it is not needed anymore, should I remove this file completely?

yes, thx

Reply:
> Execution and parameter sidenav have a lot of logic/formatting inside them, should I put something in separate files? @Redjaw

yes, but only If it is possible without complicating too much or breaking the structure. Thx